src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,47 @@
+.watch {
+  display: flex;
+  height: $watch-size;
+  width: $watch-size;
+  justify-content: center;
+  background-image: url("../images/clockface.png");
+  background-position: center;
+  background-size: cover;
+  border: $watch-border-size solid $watch-color-secondary;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    top: 12.5%;
+    height: 37.5%;
+    width: 4.5%;
+    background-color: $watch-color-main;
+    border-radius: 5px;
+    animation: time 3600s linear infinite;
+    transform-origin: bottom center;
+  }
+
+  &__seconds {
+    position: absolute;
+    top: 5%;
+    height: 45%;
+    width: 2.5%;
+    background-color: $watch-color-main;
+    border-radius: 3px;
+    animation: time 60s linear infinite;
+    transform-origin: bottom center;
+  }
+
+  &__center {
+    position: absolute;
+    top: 46.25%;
+    height: 7.5%;
+    width: 7.5%;
+    background-color: $watch-color-main;
+    border-radius: 50%;
+  }
+}
+
+.watch--position {
+  position: relative;
+  margin: auto;
+}
```

Comment:
Modifier can't set external space or position for an element. Use parent element.